packages/babel-preset-react-app/README.md

```diff
@@ -1,53 +1,4 @@
-# babel-preset-react-app
+# @fs/babel-preset-react-app
 
-This package includes the Babel preset used by [Create React App](https://github.com/facebook/create-react-app).<br>
-Please refer to its documentation:
-
-- [Getting Started](https://facebook.github.io/create-react-app/docs/getting-started) – How to create a new app.
-- [User Guide](https://facebook.github.io/create-react-app/) – How to develop apps bootstrapped with Create React App.
-
-## Usage in Create React App Projects
-
-The easiest way to use this configuration is with [Create React App](https://github.com/facebook/create-react-app), which includes it by default. **You don’t need to install it separately in Create React App projects.**
-
-## Usage Outside of Create React App
-
-If you want to use this Babel preset in a project not built with Create React App, you can install it with the following steps.
-
-First, [install Babel](https://babeljs.io/docs/setup/).
-
-Then install babel-preset-react-app.
-
-```sh
-npm install babel-preset-react-app --save-dev
-```
-
-Then create a file named `.babelrc` with following contents in the root folder of your project:
-
-```json
-{
-  "presets": ["react-app"]
-}
-```
-
-This preset uses the `useBuiltIns` option with [transform-object-rest-spread](http://babeljs.io/docs/plugins/transform-object-rest-spread/) and [transform-react-jsx](http://babeljs.io/docs/plugins/transform-react-jsx/), which assumes that `Object.assign` is available or polyfilled.
-
-## Usage with Flow
-
-Make sure you have a `.flowconfig` file at the root directory. You can also use the `flow` option on `.babelrc`:
-
-```json
-{
-  "presets": [["react-app", { "flow": true, "typescript": false }]]
-}
-```
-
-## Usage with TypeScript
-
-Make sure you have a `tsconfig.json` file at the root directory. You can also use the `typescript` option on `.babelrc`:
-
-```json
-{
-  "presets": [["react-app", { "flow": false, "typescript": true }]]
-}
-```
+This package is a fork of [babel-prese-react-app](https://github.com/facebook/create-react-app/tree/master/packages/babel-preset-react-app).<br>
+It's been specialized to the needs of Familysearch Apps
```

Comment:
```suggestion
It's been specialized to the needs of FamilySearch Apps
```

Reply:
we should include a changelog/summary of the differences from facebooks

Reply:
I understand your concerns. But I was also concerned with the fact that React's preset felt like a magical box, and that unexpected things were going on inside it. In deciding whether to use the magical box, I decided the only course of action was to sit down and understand all the things going on. It's definitely much less magical to me now, almost trivial.

These changes I'm suggesting because they make sense to me. Once I understood what jsx pragmas were all about, it felt really backward for us to be using a preset that has conflicting pragma directives in it (bolting on the emotion preset to the react preset is doing just that). 

In the end, writing a preset isn't that complicated. I feel that by writing our own, we know what we're doing.

In regards to me dropping flow and typescript: They're just as easy to add later. If we're not asking the questions about flow/typescript in the CRA tool, there's no need for them. We can very easily add that support back when the time comes, and do it in a way that makes sense for us.